backend/app/utils/config_definitions/utils.py

```
@@ -28,6 +28,11 @@
     validate_list_params,
 )
 
+from app.utils.exceptions.errors import (
+    APIError,
```

Ruff (0.8.2)

`app.utils.exceptions.errors.APIError` imported but unused

Remove unused import: `app.utils.exceptions.errors.APIError`

(F401)